expyfun/_experiment_controller.py

```diff
@@ -2182,13 +2182,10 @@ def refocus(self):
             m_hWnd = self._win._hwnd
             hCurWnd = _user32.GetForegroundWindow()
             if hCurWnd != m_hWnd:
+                _user32.SetWindowPos(m_hWnd, -1, 0, 0, 0, 0, 0x0001 | 0x0002)
```

Comment:
Let's make explicit that we're using the same thing that was commented out before
```suggestion
                # m_hWnd, HWND_TOPMOST, ..., SWP_NOSIZE | SWP_NOMOVE
                _user32.SetWindowPos(m_hWnd, -1, 0, 0, 0, 0, 0x0001 | 0x0002)
```